include/sound/sof/topology.h

```diff
@@ -53,9 +53,10 @@ struct sof_ipc_comp {
 	uint32_t id;
 	enum sof_comp_type type;
 	uint32_t pipeline_id;
+	uint32_t core;
```

Comment:
is this a linear list or a mask? e.g. how do you represent core 3?

Reply:
It's the same as for pipeline, meaning core 3 is just 3 (0 based).